src/routers/baseFieldsRouter.ts

```diff
@@ -15,5 +15,15 @@ baseFieldsRouter.put(
 	requireAdministratorRole,
 	baseFieldsHandlers.putBaseField,
 );
+baseFieldsRouter.post(
```

Comment:
I think we can (should?) actually remove the concept of POST for localizations, and ONLY have `PUT`.

You would then change PUT to up-sert instead of simply update.  This wouldn't violate rest because the action is still idempotent.  Since language is part of the primary key this also makes your life easier -- it's a bit odd to include primary key information in the body of a POST or PUT and most of the tooling is build around preventing that kind of thing.